sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,140 @@
+# tag_based_pnp_calibrator
+
+In this tutorial, we will present a hands-on tutorial of the `tag_based_pnp_calibrator`. Although we provide pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
```

Comment:
The tense should be infinitive  here

Reply:
@knzo25 
Sorry, I am not sure where to modify it. 
btw, it is the same context as the sfm one.

Reply:
@vividf 
Nice, fixed the sfm one in
d1a3bd2